src/arrays.js

```diff
@@ -1,59 +1,164 @@
 export const getNthElement = (index, array) => {
-  // your code here
+  
+if(index >= array.length) {
+  
```

Comment:
Never have line breaks directly after a block of code starts, or right before it ends.